distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -1018,4 +1023,16 @@ public String getOption() {
       }
     }
   }
+
+  private static class TestMethodTimer extends Stopwatch {
+    @Override
+    protected void succeeded(long nanos, Description description) {
+      if (LOGGER.isDebugEnabled()) {
+        LOGGER.debug(
```

Comment:
✏️ Might be useful to print out the `description.getClassName()` here as well since lots of tests extend this class.

Reply:
Done